web_accounts_list.json

```diff
@@ -372,6 +372,17 @@
          "category" : "video",
          "valid" : true
       },
+      {
+         "name" : "Dribbble",
+         "check_uri" : "https://dribbble.com/{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : " | Dribbble\n",

```

Comment:
I'm concerned about the \n in the string here. From what I'm seeing, it is not necessary for the matching and it could cause issues downstream. Can you check this please and adjust?